clover/__init__.py

```diff
@@ -75,15 +76,22 @@ def __init__(self, name, filename, statements, covered_statements,
         self.covered_conditions = covered_conditions
         self.ncloc = statements
 
-    def count_loc(self):
+    def count_loc(self, sources):
+        if not sources:
+            # Old version of coverage
+            sources = ['']
         i = 0
-        try:
-            with open(self.filename) as f:
-                for i, l in enumerate(f):
-                    pass
-        except IOError:
-            self.loc = 0
-        self.loc = i + 1
+        for source in sources:
+            filename = os.path.join(source, self.filename)
+            try:
+                with open(filename) as f:
+                    for i, __ in enumerate(f, start=1):
```

Comment:
may be `max(loc)` for each filename should be computed here? Now `i` is just number of lines in last opened file

Reply:
oops, only first existing file is opened (loop break below) - so, need some comments about what's going on here :-)

Reply:
@tumb1er, added some comments.